abl-schema.json

```diff
@@ -2,6 +2,9 @@
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
   "properties": {
+    "api_version": {
+      "type": "number"
```

Comment:
Do you know if JSONSchema allows us to switch schemas on this field?

Reply:
Sorry about the late reply. I do not know if you can use a different schema based on a field within a document. At least, not within JSONSchema itself. I will do some searching on this. 

You could also consider looking at the value manually and using a different schema based on the value.

Reply:
I forgot, I used something like this to implement some card validation once:
[json-schema if-then-else](http://json-schema.org/understanding-json-schema/reference/conditionals.html#if-then-else)

It's not exactly using an entirely different schema, but you could conditionally change parts of a schema based on the api_version value.